extensions-contrib/graphite-emitter/src/main/java/io/druid/emitter/graphite/GraphiteEmitter.java

```diff
@@ -92,7 +91,8 @@ public void start()
   public void emit(Event event)
   {
     if (!started.get()) {
-      throw new ISE("WTF emit was called while service is not started yet");
+      log.error("WTF emit was called while service is not started yet");
```

Comment:
I think this exception is actually fine, since it's a violation of expected usage. HttpPostEmitter throws an exception in a similar situation. The only exceptions that are really "dangerous" here are ones that could occur during normal usage.